bccsp/sw/fileks.go

```diff
@@ -219,7 +232,7 @@ func (ks *fileBasedKeyStore) searchKeystoreForSKI(ski []byte) (k bccsp.Key, err
 
 		return k, nil
 	}
-	return nil, fmt.Errorf("Key with SKI %s not found in %s", hex.EncodeToString(ski), ks.path)
+	return nil, fmt.Errorf("key with SKI %s not found in %s", hex.EncodeToString(ski), ks.path)
 }
```

Comment:
Just use %x to format as hex instead of using the hex package.

Reply:
addressed in #936